sprints/webhooks/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+import requests
+import json
```

Comment:
Please sort imports as [defined is PEP-8](https://www.python.org/dev/peps/pep-0008/#imports):
```python
import json

import requests
from django.db import models
```